internal/resources/storage_statefulset.go

```diff
@@ -182,6 +187,57 @@ func (b *StorageStatefulSetBuilder) buildContainer() corev1.Container {
 	return container
 }
 
+func (b *StorageStatefulSetBuilder) buildVolumeMounts() []corev1.VolumeMount {
+	volumeMounts := []corev1.VolumeMount{
+		{
+			Name:      configVolumeName,
+			ReadOnly:  true,
+			MountPath: "/opt/kikimr/cfg",
+		},
+	}
+
+	if b.Spec.Service.GRPC.TLSConfiguration.Enabled {
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      "grpc-tls-volume",
+			ReadOnly:  true,
+			MountPath: "/tls/grpc",
+		})
+	}
+
+	if b.Spec.Service.Interconnect.TLSConfiguration.Enabled {
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      "interconnect-tls-volume",
+			ReadOnly:  true,
+			MountPath: "/tls/interconnect",
+		})
+	}
+
+	return volumeMounts
+}
+
+func (b *StorageStatefulSetBuilder) buildContainerArgs() ([]string, []string) {
+	command := []string{"/opt/kikimr/bin/start.sh"}
+	var args []string
+
+	if b.Spec.Service.Interconnect.TLSConfiguration.Enabled {
+		args = append(args,
+			//"--ca",
+			//"/tls/interconnect/ca.crt",  // re-check, may be unnecessary, also overrides gRPC CA
+			"--cert",
+			"/tls/interconnect/tls.crt",
+			"--key",
+			"/tls/interconnect/tls.key",
+		)
+	}
+
+	if b.Spec.ClusterConfig != "" {
+		command = []string{"/bin/bash"}
+		args = append(args, "-c", "source /opt/kikimr/cfg/kikimr.cfg && exec /opt/kikimr/bin/kikimr ${kikimr_arg}")
```

Comment:
here you are using --cert and --key args with /bin/bash which will cause "invalid option" error